packages/plugin-js-packages/src/lib/runner/audit/transform.unit.test.ts

```diff
@@ -271,16 +280,17 @@ describe('vulnerabilitiesToIssues', () => {
       vulnerabilitiesToIssues(
         [
           {
+            ...vulnerabilityDefaults,
             name: '@cypress/request',
             directDependency: 'cypress',
           },
-        ] as Vulnerability[],
+        ],
         defaultAuditLevelMapping,
       ),
     ).toEqual<Issue[]>([
       expect.objectContaining({
         message: expect.stringContaining(
-          "`cypress`' dependency `@cypress/request` has",
+          "`cypress`'s dependency `@cypress/request` has",
```

Comment:
🐞 This was the main added value of `apostrophize`- it could tell when the word ended with an "s" so that it would not add another "s" after the apostrophe. The current result is grammatically incorrect.

Reply:
I know, but I deliberately removed this logic because it was both over-complicated and unnecessary. 

Grammatically, "`cypress`'s" is perfectly acceptable. In fact, both `s's` and `s'` forms are in common use. There is no hard rule about when to use which, just [stylistic preferences](https://www.grammarly.com/blog/possessive-case/#:~:text=This%20general%20rule,be%20used%20consistently). The only differentiation (loosely upheld) I've come across is [`s's` for singular nouns](https://www.scribbr.com/language-rules/apostrophe-s/#:~:text=Names%20ending%20in,is%20always%20wrong.) and `s'` for plural nouns. And NPM package name's are predominantly singular.

Reply:
I see. Not a fan of this change but thank you for the explanation, seems legit.

Reply:
Removed it in 9d0947518c5d848dd003bba411cceac78308e8d7